scripts/ci_build.sh

```diff
@@ -28,18 +28,18 @@ if [ -z "$HOME" ]; then
     mkdir -p $HOME
 fi
 
+# Set Docker Container name to be unique
+container_name=""
 
+#Jenkins
+[ ! -z "$BUILD_TAG" ] && container_name="$BUILD_TAG"
+#VSO
+[ ! -z "$BUILD_BUILDID" ] && container_name="$BUILD_BUILDID"
 
-if [[ "$OSNAME" == "ubuntu" ]]; then
-    # Set Docker Container name to be unique
-    container_name=""
+export DOTNET_BUILD_CONTAINER_NAME="$container_name"
```

Comment:
ci_build.sh is currently run only by VSO/Jenkins. It must not be used for building in dev boxes. Even then we fall back to a [default container name](https://github.com/dotnet/cli/blob/master/scripts/docker/dockerbuild.sh#L22).